app/assets/javascripts/components/search/SearchResults.js

```diff
@@ -14,6 +14,7 @@ export const SearchResults = ({ isEnabled, searchResults, closeSearch, projectBo
   return (
     <Column
       onClose={closeSearch}
+      canCloseColumn={true}
```

Comment:
Since it's a prop with the value `true`, you can omit the value (just like it's possible in HTML):

```suggestion
      canCloseColumn
```

Reply:
Didn't now that :thinking: , thks.